webviz_config/plugins/_utils.py

```diff
@@ -39,20 +39,34 @@ def load_webviz_plugins_with_metadata(
                 }
 
                 if entry_point.name in metadata:
+                    if metadata[entry_point.name] == PluginDistInfo(
+                        {
+                            "dist_name": dist.metadata["name"],
+                            "dist_version": dist.version,
+                            "documentation_url": project_urls.get("Documentation"),
+                            "download_url": project_urls.get("Download"),
+                            "tracker_url": project_urls.get("Tracker"),
+                        }
+                    ):
+                        # May occur if folders in virtual environment are symlinked.
+                        continue
+
                     warnings.warn(
                         f"Multiple versions of plugin with name {entry_point.name}. "
                         f"Already loaded from project {metadata[entry_point.name]['dist_name']}. "
                         f"Overwriting using plugin with from project {dist.metadata['name']}",
                         RuntimeWarning,
                     )
 
-                metadata[entry_point.name] = {
-                    "dist_name": dist.metadata["name"],
-                    "dist_version": dist.version,
-                    "documentation_url": project_urls.get("Documentation"),
-                    "download_url": project_urls.get("Download"),
-                    "tracker_url": project_urls.get("Tracker"),
-                }
+                metadata[entry_point.name] = PluginDistInfo(
+                    {
+                        "dist_name": dist.metadata["name"],
+                        "dist_version": dist.version,
+                        "documentation_url": project_urls.get("Documentation"),
+                        "download_url": project_urls.get("Download"),
+                        "tracker_url": project_urls.get("Tracker"),
+                    }
+                )

```

Comment:
Define this metadata dictionary once as a variable? Here it is introduced two times (more work to maintain)?

Reply:
I suggest just leaving the duplication for now. My upcoming and revised https://github.com/equinor/webviz-config/pull/377 ends up rewriting/moving this code so the duplication can removed then.